eng/pipelines/common/templates/runtimes/send-to-helix-step.yml

```diff
@@ -82,7 +82,7 @@ steps:
       condition: ${{ parameters.condition }}
     env:
       __BuildArch: ${{ parameters.archType }}
-      __TargetOS: ${{ parameters.osGroup }}${{ parameters.osSubGroup }}
+      __TargetOS: ${{ parameters.osGroup }}${{ parameters.osSubgroup }}
```

Comment:
Looks good. At some point we may want to consider splitting these too into something like __TargetOSGroup / Subgroup but please avoid doing that as part of this change :-).